cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -937,15 +937,38 @@ a70175636f61703a2f2f61732e6578616d706c652e636f6d02656572696b7703
 	  Line breaks are for display purposes only.
 	</t>
 
-	<section anchor="exampleKeysSymmetric128" title='128-bit Symmetric Key as Hex Encoded String'>
-	  <figure><artwork><![CDATA[
-231f4c4d4d3051fdc2ec0a3851d5b383
+	<section anchor="exampleKeysSymmetric128" title='128-bit Symmetric Key'>
+    <figure align="center" anchor="fig:exampleKeysShared128Hex"
+title='128-bit symmetric COSE key as hex string'><artwork align="left"><![CDATA[
+a42050231f4c4d4d3051fdc2ec0a3851d5b3830104024c53796d6d6574726963
+313238030a
+]]></artwork></figure>
+<figure align="center" anchor="fig:exampleKeysShared128Diag"
+title='128-bit symmetric COSE key in CBOR diagnostic notation'><artwork align="left"><![CDATA[
+{
+  / k /   -1: h’231f4c4d4d3051fdc2ec0a3851d5b383’
+  / kty /  1: 4 / Symmetric /,
+  / kid /  2: 'Symmetric128',
```

Comment:
COSE Key IDs are byte sequences.  This looks like a text string to me.

Reply:
Yes it does look like a text string. 
I looked at the COSE examples to see how Jim displayed it and he did like this (https://tools.ietf.org/html/rfc8152#appendix-C).

If you decode the hex string if is encoded as byte string

It could be that Jim got it wrong in the COSE doc, or that I´m missing something

Reply:
Maybe a question to Carsten is in place